### DISCUSSION TITLE
23:33:55-512467 WARNING  Using SDXL VAE loaded from singular file will result in low contrast images.

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [vladmandic/sdnext#1965] [Q&A]
Sorry, I keep spamming my questions, but I doubt I can get my questions solved from any other place.

**Comment by bmaxdk** [comment]
You may want to check the way the images are preprocess before being fed into the model. Also If the model was trained on a dataset with low contrast or different contrast characteristics, it may produce low contrast outputs

**Comment by vladmandic** [comment]
That was the issue and warning was valid, but it was resolved - now VAEs for SDXL work fine.
